scripts/start-mspass.sh

```diff
@@ -98,5 +157,4 @@ if [ $# -eq 0 ]; then
   fi
 else
   docker-entrypoint.sh $@
-fi
-
+fi
```

Comment:
Just a note here, it is preferred to always have a newline at the end of any files.